File: Assignment 16/Question3.java
import java.util.Scanner;

class Factorial 
{
    public int iCnt = 0;
    public int iFactorial = 1;

    public int Factor(int iNo)
    {
        for (iCnt = 1 ; iCnt<=iNo ; iCnt++)
        {
            iFactorial = iCnt * iFactorial ;
        }
        return iFactorial;
    }
}
class Question3 
{
    public static void main(String[] args) 
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0 ;
        int iRet = 0 ;

        System.out.println("Enter your Number :");
        iValue = sobj.nextInt();

        Factorial fobj = new Factorial();

        iRet = fobj.Factor(iValue);

        System.out.println("The factorial is :"+  iRet);




    }
    
}

File: Assignment 8/Question3.c
 #include <stdio.h>

int Factorial(int iNo)
{

    int iCnt = 0 , iFactor = 0 ;
    iFactor = 1;

    //UPDATER
    if ( iNo <0)
    {
        iNo = -iNo ;
    }
    for (iCnt = 1 ; iCnt<=iNo;iCnt++)
    {
        iFactor = iFactor *iCnt ;
    }
    return iFactor ;
}
int main()
{
    int iValue = 0 , iRet = 0;
    printf("Enter Number: ");
    scanf("%d",&iValue);

    iRet = Factorial(iValue);

    printf("%d is the factorial of %d", iRet, iValue);
    return 0 ;
}
